assets/apps/dashboard/src/Components/Content/CustomLayoutsUnavailable.js

```diff
@@ -0,0 +1,52 @@
+/* global neveDash */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+
+const CustomLayoutsUnavailable = (props) => {
+	const { setTab } = props;
+	const { customLayoutsNeveProURL, assets } = neveDash;
+
+	return (
+		<div className="unavailable-custom-layouts">
+			<div
+				className="ss-background"
+				style={{ backgroundImage: `url(${assets}/starter.jpg)` }}
+			/>
+			<div className="content-wrap">
+				<img src={`${assets}/logo.svg`} alt="Neve" />
+				<h1>{__('Custom Layouts', 'neve')}</h1>
+				<p>
+					{__(
+						'Get access to all Pro features and power-up your website',
+						'neve'
+					)}
+				</p>
+				<iframe
+					width="560"
+					height="315"
+					src="https://www.youtube.com/embed/4XFoSHTHpes"
+					title="YouTube video player"
+					frameBorder="0"
+					allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+					allowFullScreen
+				></iframe>
+				<Button
+					target="_blank"
+					rel="external noreferrer noopener"
+					href={customLayoutsNeveProURL}
+					isPrimary
```

Comment:
It's a nit, isPrimary has been deprecated while back and will be removed at some point from codebase, we can use variant="primary" as a prop instead.

```suggestion
					variant="primary"
```